schema.graphql

```diff
@@ -646,7 +646,7 @@ enum Frequency {
 
 type Fund {
   _id: ID!
-  campaign: [FundraisingCampaign]!
+  campaigns: [FundraisingCampaign]!
```

Comment:
Funds may or may not have associated campaigns. Please fix and adjust your tests.

Reply:
fixed it .